<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewRecordTable</class>
 <widget class="QWidget" name="NewRecordTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1680</width>
    <height>910</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#NewRecordTable{background: #FFFFFF;}</string>
  </property>
  <widget class="QPushButton" name="evaluationRecord_Btn">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>40</y>
     <width>150</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #EFEFEF;
border-radius: 4px;
color:#333333;</string>
   </property>
   <property name="text">
    <string>评估记录</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="trainRecord_Btn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>150</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #0D9DDB;
border-radius: 4px;
color:white;</string>
   </property>
   <property name="text">
    <string>训练记录</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>1551</width>
     <height>711</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#stackedWidget{background: #FFFFFF;}</string>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="trainRecord_Page">
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1551</width>
       <height>46</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">width: 1592px;
height: 46px;
background: #F1F9FD;
border:none;</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>11</y>
        <width>111</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">width: 96px;
height: 24px;
font-size: 24px;
font-family: SourceHanSansCN-Regular, SourceHanSansCN;
font-weight: 400;
color: #676767;
line-height: 36px;</string>
      </property>
      <property name="text">
       <string>训练类型</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>11</y>
        <width>101</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">width: 96px;
height: 24px;
font-size: 24px;
font-family: SourceHanSansCN-Regular, SourceHanSansCN;
font-weight: 400;
color: #676767;
line-height: 36px;</string>
      </property>
      <property name="text">
       <string>训练日期</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>572</x>
        <y>11</y>
        <width>161</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">width: 96px;
height: 24px;
font-size: 24px;
font-family: SourceHanSansCN-Regular, SourceHanSansCN;
font-weight: 400;
color: #676767;
line-height: 36px;</string>
      </property>
      <property name="text">
       <string>训练时长(min)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>11</y>
        <width>51</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">width: 96px;
height: 24px;
font-size: 24px;
font-family: SourceHanSansCN-Regular, SourceHanSansCN;
font-weight: 400;
color: #676767;
line-height: 36px;</string>
      </property>
      <property name="text">
       <string>游戏</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>970</x>
        <y>11</y>
        <width>51</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">width: 96px;
height: 24px;
font-size: 24px;
font-family: SourceHanSansCN-Regular, SourceHanSansCN;
font-weight: 400;
color: #676767;
line-height: 36px;</string>
      </property>
      <property name="text">
       <string>得分</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>1090</x>
        <y>11</y>
        <width>51</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">width: 96px;
height: 24px;
font-size: 24px;
font-family: SourceHanSansCN-Regular, SourceHanSansCN;
font-weight: 400;
color: #676767;
line-height: 36px;</string>
      </property>
      <property name="text">
       <string>查看</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>1220</x>
        <y>11</y>
        <width>61</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">width: 96px;
height: 24px;
font-size: 24px;
font-family: SourceHanSansCN-Regular, SourceHanSansCN;
font-weight: 400;
color: #676767;
line-height: 36px;</string>
      </property>
      <property name="text">
       <string> 打印</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>1370</x>
        <y>11</y>
        <width>51</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">width: 96px;
height: 24px;
font-size: 24px;
font-family: SourceHanSansCN-Regular, SourceHanSansCN;
font-weight: 400;
color: #676767;
line-height: 36px;</string>
      </property>
      <property name="text">
       <string>删除</string>
      </property>
     </widget>
    </widget>
    <widget class="QTableWidget" name="train_TableWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>1551</width>
       <height>560</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>14</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLineEdit" name="trainDestPage_LineEdit">
     <property name="geometry">
      <rect>
       <x>1323</x>
       <y>646</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">width: 90px;
height: 50px;
background: #FFFFFF;
border-radius: 4px;
border: 1px solid #EBEBEB;</string>
     </property>
     <property name="text">
      <string>1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="trainIndexPage_Label">
     <property name="geometry">
      <rect>
       <x>943</x>
       <y>654</y>
       <width>120</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">width: 47px;
height: 22px;
font-size: 22px;
font-family: SourceHanSansCN-Normal, SourceHanSansCN;
font-weight: 400;
color: #676767;
line-height: 33px;</string>
     </property>
     <property name="text">
      <string>第1/20页</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="trainGo_btn">
     <property name="geometry">
      <rect>
       <x>1473</x>
       <y>640</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">width: 32px;
height: 22px;
font-size: 22px;
font-family: SourceHanSansCN-Normal, SourceHanSansCN;
font-weight: 400;
color: #676767;
line-height: 33px;
width: 50px;
height: 50px;
background: #FFFFFF;
border-radius: 4px;
border: 1px solid #EBEBEB;</string>
     </property>
     <property name="text">
      <string>GO</string>
     </property>
    </widget>
    <widget class="QPushButton" name="trainFrontPage_btn">
     <property name="geometry">
      <rect>
       <x>1083</x>
       <y>640</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>1420</x>
       <y>650</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">width: 22px;
height: 22px;
font-size: 22px;
font-family: SourceHanSansCN-Normal, SourceHanSansCN;
font-weight: 400;
color: #676767;
line-height: 33px;</string>
     </property>
     <property name="text">
      <string>页</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>1250</x>
       <y>654</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">width: 66px;
height: 22px;
font-size: 22px;
font-family: SourceHanSansCN-Normal, SourceHanSansCN;
font-weight: 400;
color: #676767;
line-height: 33px;</string>
     </property>
     <property name="text">
      <string>跳转到</string>
     </property>
    </widget>
    <widget class="QPushButton" name="trainBackPage_btn">
     <property name="geometry">
      <rect>
       <x>1163</x>
       <y>640</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(85, 255, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="evaluateRecord_Page">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1551</width>
       <height>46</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">width: 1592px;
height: 46px;
background: #F1F9FD;
border:none;</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>11</y>
        <width>111</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">width: 96px;
height: 24px;
font-size: 24px;
font-family: SourceHanSansCN-Regular, SourceHanSansCN;
font-weight: 400;
color: #676767;
line-height: 36px;</string>
      </property>
      <property name="text">
       <string>评估类型</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>11</y>
        <width>121</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">width: 96px;
height: 24px;
font-size: 24px;
font-family: SourceHanSansCN-Regular, SourceHanSansCN;
font-weight: 400;
color: #676767;
line-height: 36px;</string>
      </property>
      <property name="text">
       <string> 评估日期</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>11</y>
        <width>161</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">width: 96px;
height: 24px;
font-size: 24px;
font-family: SourceHanSansCN-Regular, SourceHanSansCN;
font-weight: 400;
color: #676767;
line-height: 36px;</string>
      </property>
      <property name="text">
       <string>评估时长(min)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>1000</x>
        <y>10</y>
        <width>111</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">width: 96px;
height: 24px;
font-size: 24px;
font-family: SourceHanSansCN-Regular, SourceHanSansCN;
font-weight: 400;
color: #676767;
line-height: 36px;</string>
      </property>
      <property name="text">
       <string>评估范围</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>1220</x>
        <y>10</y>
        <width>80</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">width: 96px;
height: 24px;
font-size: 24px;
font-family: SourceHanSansCN-Regular, SourceHanSansCN;
font-weight: 400;
color: #676767;
line-height: 36px;</string>
      </property>
      <property name="text">
       <string>删除</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QTableWidget" name="evaluate_TableWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>1551</width>
       <height>560</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>14</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>1430</x>
       <y>650</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">width: 22px;
height: 22px;
font-size: 22px;
font-family: SourceHanSansCN-Normal, SourceHanSansCN;
font-weight: 400;
color: #676767;
line-height: 33px;</string>
     </property>
     <property name="text">
      <string>页</string>
     </property>
    </widget>
    <widget class="QPushButton" name="evaluateGo_btn">
     <property name="geometry">
      <rect>
       <x>1470</x>
       <y>640</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">width: 32px;
height: 22px;
font-size: 22px;
font-family: SourceHanSansCN-Normal, SourceHanSansCN;
font-weight: 400;
color: #676767;
line-height: 33px;
width: 50px;
height: 50px;
background: #FFFFFF;
border-radius: 4px;
border: 1px solid #EBEBEB;</string>
     </property>
     <property name="text">
      <string>GO</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="evaluateDestPage_LineEdit">
     <property name="geometry">
      <rect>
       <x>1330</x>
       <y>645</y>
       <width>91</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">width: 90px;
height: 50px;
background: #FFFFFF;
border-radius: 4px;
border: 1px solid #EBEBEB;</string>
     </property>
     <property name="text">
      <string>1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="evaluateIndexPage_Label">
     <property name="geometry">
      <rect>
       <x>960</x>
       <y>654</y>
       <width>100</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">width: 47px;
height: 22px;
font-size: 22px;
font-family: SourceHanSansCN-Normal, SourceHanSansCN;
font-weight: 400;
color: #676767;
line-height: 33px;</string>
     </property>
     <property name="text">
      <string>1/23页</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>1242</x>
       <y>653</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">width: 66px;
height: 22px;
font-size: 22px;
font-family: SourceHanSansCN-Normal, SourceHanSansCN;
font-weight: 400;
color: #676767;
line-height: 33px;</string>
     </property>
     <property name="text">
      <string>跳转到</string>
     </property>
    </widget>
    <widget class="QPushButton" name="evaluationFrontPage_btn">
     <property name="geometry">
      <rect>
       <x>1080</x>
       <y>640</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="evaluationBackPage_btn">
     <property name="geometry">
      <rect>
       <x>1160</x>
       <y>640</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
